fleetctl/fleetctl.go

```diff
@@ -732,6 +803,64 @@ func lazyCreateUnits(args []string) error {
 	return nil
 }
 
+// matchUnitFiles compares two unitFiles
+// Returns true if the units match, false otherwise.
+func matchUnitFiles(a *unit.UnitFile, b *unit.UnitFile) bool {
+	if a.Hash() == b.Hash() {
+		return true
+	}
+
+	return false
+}
+
+// matchLocalFileAndUnit compares a file with a Unit
+// Returns true if the contents of the file matches the unit one, false
+// otherwise; and any error ocountered
+func matchLocalFileAndUnit(file string, su *schema.Unit) (bool, error) {
+	result := false
+	a := schema.MapSchemaUnitOptionsToUnitFile(su.Options)
+
+	_, err := os.Stat(file)
+	if err == nil {
+		b, err := getUnitFromFile(file)
```

Comment:
@dongsupark the mentioned function existed in the original code: https://github.com/coreos/fleet/blob/master/fleetctl/fleetctl.go#L729 ; part if it's logic ended up here -- but it was altered in the process. Specifically, the original had explicit handling for `os.IsNotExist(err)`.